experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/src/OTLPMetricExporter.ts

```diff
@@ -33,7 +32,7 @@ import { createExportMetricsServiceRequest, IExportMetricsServiceRequest } from
 
 class OTLPMetricExporterProxy extends OTLPGRPCExporterNodeBase<ResourceMetrics, IExportMetricsServiceRequest> {
 
-  constructor(config: OTLPGRPCExporterConfigNode & OTLPMetricExporterOptions= defaultOptions) {
+  constructor(config?: OTLPGRPCExporterConfigNode & OTLPMetricExporterOptions) {
```

Comment:
It looks like `defaultOptions` is no longer been used. Should we remove the definition of it?

Reply:
Jep, it is no longer used anymore :slightly_smiling_face: 
It is already removed in the current diff AFAICT :slightly_smiling_face:

Reply:
Oh, yeah, I might have somehow looked over that part.